src/renderServer.ts

```diff
@@ -60,7 +65,11 @@ export function renderServer(config: ConfigService) {
     });
   });
 
-  app.get('/health-check', (_req, resp) => resp.status(200).send('OK'));
+  app.get('/health-check', (_req, resp) =>
+    config.isLoaded
+      ? resp.status(200).send('OK')
+      : resp.status(503).send('NOT CONFIGURED')
```

Comment:
Is this safe? Example scenario: we deploy this behind GCP load balancer, it gets the `503` 3 times in a row and then restarts the service to fix the issue (or takes it offline).

If this scenario makes sense, then this change is OK by me. I would prefer/expect a safe fallback instead tho.

Reply:
> Is this safe? Example scenario: we deploy this behind GCP load balancer, it gets the 503 3 times in a row and then restarts the service to fix the issue (or takes it offline).

I wonder if we'll need to have a different status code. Something to be discussed with ops for sure.

cc @jasonious for some visability here